src/python/bot/tasks/fuzz_task.py

```diff
@@ -990,8 +996,7 @@ def create_testcase(group, context):
     testcase.put()
 
   fuzzing_strategies = (
```

Comment:
So this cf::fuzzing_strategies thing is a holdout from our old (deprecated) way of piping strategies. We still need this support this for the time being until AFL is migrated. In the new pipeline, we already have strategies available without needing to parse them. 

Let's move this to the `from_testcase_manager_crash` constructor (which is what the deprecated way is using), and make `fuzzing_strategies` an instance variable of `Crash`. Then, we can do something like this:

```python
  
def run_engine_fuzzer(...):
   ...
   return result, fuzzer_metadata, options.strategies

def do_engine_fuzzing(...):
  ... pipe strategies from run_engine_fuzzer and return here...
   
  @classmethod
  def from_testcase_manager_crash(...):
    fuzzing_strategies = (
        libfuzzer_stats.LIBFUZZER_FUZZING_STRATEGIES.search(orig_unsymbolized_crash_stacktrace))
    return Crash( 
        ...
        fuzzing_strategies=fuzzing_strategies)

  def from_engine_crash(cls, crash, fuzzing_strategies):
    return Crash(
      ...
      fuzzing_strategies=fuzzing_strategies)

  # Crash class
  def __init__(self,
               file_path,
               crash_time,
               return_code,
               resource_list,
               gestures,
               unsymbolized_crash_stacktrace,
               arguments,
               application_command_line,
               http_flag=False,
               fuzzing_strategies=None):
  ...
  self.fuzzing_strategies = fuzzing_strategies
```

Does this make sense?